manager/assets/modext/widgets/element/modx.panel.tv.js

```diff
@@ -1399,6 +1400,9 @@ Ext.extend(MODx.panel.TVInputProperties,MODx.Panel,{
                 */
                 if (typeChanged) {
                     Ext.getCmp('modx-tv-elements').clearInvalid();
+                    if(this.isNativeType){
+                        inputOptValsItem.allowBlank = false;
+                    }
```

Comment:
Out of curiosity:

What happens if you change the type, this statement is triggered because it's a nativeType and you switch the type again to a type that's not a native type? Will it switch back to `inputOptValsItem.allowBlank = true` somewhere?

Reply:
The `allowBlank` property is only automatically set internally for native types, so there should be no problem with non-native ones. I believe I checked that when testing, but will double-check to be sure.

Reply:
Yup - confirmed - everything works as expected when switching from native to non-native types.